modules/xfem/src/base/XFEM.C

```diff
@@ -893,7 +893,7 @@ XFEM::cutMeshWithEFA()
 
     new_node->set_n_systems(parent_node->n_systems());
     efa_id_to_new_node.insert(std::make_pair(new_node_id,new_node));
-    _console<<"XFEM added new node: "<<new_node->id()+1<<std::endl;
+    _console << "XFEM added new node: " << new_node->id() <<std::endl;
```

Comment:
Time to start getting rid of these overused `std::endls`. Generally you should do a bunch of output then a single flush. This is inside of a for loop so it's just unecessary and potentially slow.